misc/genstd/exprstring.go

```diff
@@ -0,0 +1,289 @@
+// Forked from go/types (go 1.20.3) to implement support for *linkedIdent.
+
+// Copyright 2013 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file implements printing of expressions.
```

Comment:
Do you know what part of this file is not copied? Can we split the copied part from the added one?

Reply:
diff: https://www.diffchecker.com/g9A1izhf/

I would like to say yes, but essentially 1. I changed the function to be a method of `exprPrinter`, in order to have different "printing strategies" for where ExprString is used (the function is very recursive, so this changes all the recursive calls in the switch-case as well), so this would need to stay in the copied part. 2. the only way I can split this, I believe, is if I take out the case for `*linkedIdent` in the type switch, add that to a different function, and call this function before the big type-switch (if it returns true, then the caller returns). IMO it's not worth it, wdyt?

Reply:
maybe we should add this as a comment for posterity